src/engine/strat_engine/backstore/metadata/bda.rs

```diff
@@ -270,6 +223,53 @@ impl StaticHeader {
         }
     }
 
+    /// Read the BDA from the device and return 2 SECTORS worth of data, one for each BDA returned
```

Comment:
Please add an additional commit to change the header comment to something like this:

/// Read the data at both signature block locations.
/// Return the data from each location as an array of bytes
/// or an error if the read fails. Return the values
/// in the order in which they occur on the device.
/// Read the contents of each signature block separately,
/// as this increases the probability that at least one read
/// will not fail.